src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        Random random = new Random();
+        return COLORS[random.nextInt(COLORS.length)];
```

Comment:
because we won't reuse `random`, it is possible not to create this variable
```suggestion
        return COLORS[new Random().nextInt(COLORS.length)];
```